firedrake/mesh.py

```diff
@@ -635,31 +636,33 @@ class QuadrilateralMesh(MeshBase):
     """
 
     @utils.cached_property
-    def _closure_ordering(self):
-        """Pair of the cell closure and edge directions."""
-        return dmplex.quadrilateral_closure_ordering(self._plex, self._cell_numbering)
-
-    @property
     def cell_closure(self):
         """2D array of ordered cell closures
 
         Each row contains ordered cell entities for a cell, one row per cell.
         """
-        return self._closure_ordering[0]
 
-    def create_cell_node_list(self, global_numbering, fiat_element, dofs_per_cell):
-        """Builds the DoF mapping.
+        plex = self._plex
+        cell_numbering = self._cell_numbering
 
-        :arg global_numbering: Section describing the global DoF numbering
-        :arg fiat_element: The FIAT element for the cell
-        :arg dofs_per_cell: Number of DoFs associated with each mesh cell
-        """
-        edge_directions = self._closure_ordering[1]
-        return dmplex.get_quadrilateral_cell_nodes(global_numbering,
-                                                   self.cell_closure,
-                                                   edge_directions,
-                                                   fiat_element,
-                                                   dofs_per_cell)
+        # Global vertex numbering
+        vertex_numbering = plex.getDefaultGlobalSection()
```

Comment:
This will only be the vertex numbering if the last call to `plex.setDefaultSection` was made using a section with 1 dof on each vertex and nothing else.  Is this always guaranteed to be the case?